notebooks/text_models/solutions/load_text.ipynb

```diff
@@ -0,0 +1,1935 @@
+{
```

Comment:
<p>We don't need to specify shuffle=False since we are using a seed. But if we don't use a seed then we have to specify <code>shuffle=False</code>.  <span class="ql-cursor">﻿</span></p>